src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -201,6 +198,12 @@ export const ArtistSeriesHeaderFragmentContainer = createFragmentContainer(
         artworksCountMessage
         descriptionFormatted(format: HTML)
         image {
+          _xs: cropped(height: 360, width: 360, version: "large") {
+            url
+          }
+          _sm: resized(width: 1200, version: "normalized") {
```

Comment:
**Non-blocking**: FWIW I only prefix the version with an underscore because typically they are 1x/2x and you can't have a number begin a key.

Reply:
Ahhhhh I should have asked! That makes sense.